doc/UsersGuide/BuildingRunningTesting/tutorials/tutorial6.inc

```diff
@@ -0,0 +1,667 @@
+
+When modeling severe weather, it is preferable to use a high-resolution (~1-4km) grid along with physics suites and data designed for these convection-allowing scales. Therefore, High-Resolution Rapid Refresh (HRRR) data were downloaded for this case. According to NOAA, "`HRRR <https://rapidrefresh.noaa.gov/hrrr/>`_ is a NOAA real-time 3-km resolution, hourly updated, cloud-resolving, convection-allowing atmospheric model, initialized by 3km grids with 3km radar assimilation."
+
+To download data for this case, users can create a directory named ``data`` (in ``$GCB`` or an equivalent directory) and place the ``get_data.py`` script in there:
+
+.. code-block:: console
+
+   mkdir data
+   cd data
+
+Users can download the ``get_data.py`` :srw-wiki:`utility script <Tutorial/get_data.py>` with ``wget`` or copy-paste the contents of the script into a file on their system to download recent HRRR data from the `Planetary Computer <https://planetarycomputer.microsoft.com/dataset/storage/noaa-hrrr>`_ site. 
+
+.. code-block:: console
+
+   wget https://github.com/ufs-community/ufs-srweather-app/wiki/Tutorial/get_data.py
+
+.. note::
+
+   Users will need at least Python 3.10 and the ``requests`` library to run the script as-is. 
+
+Call the script with the ``-h`` option to see usage information: 
+
+.. code-block:: console
+
+   $ python3 get_data.py -h
+   usage: get_data.py [-h] --date DATE --product
+                   {wrfsfcf,wrfprsf,wrfnatf,wrfsubhf} [--fhour [FH ...]]
+
+   Utility for retrieving HRRR data
+
+   options:
+   -h, --help            show this help message and exit
+   --date DATE, -d DATE  Date of requested data in YYYYMMDDHH format
+   --product {wrfsfcf,wrfprsf,wrfnatf,wrfsubhf}, -p {wrfsfcf,wrfprsf,wrfnatf,wrfsubhf}
+                           Product type (e.g., "wrfsfcf", "wrfprsf")
+   --fhour [FH ...], -f [FH ...]
+                           Forecast hours
+
+Users are welcome to adapt the script to work for other data sources, such as RAP or GFS data. 
+
+When creating this experiment, the following command was used to download the data:
+
+.. code-block:: console
+
+   python3 get_data.py -p wrfprsf -d 2025012100 -f 0 3 6 9 12 15 18 21 24 27 30 33 36
+
+Users will need to adjust this command for the specific date(s) and forecast hours they need for their experiment. Although the Gulf Coast Blizzard data is no longer available via Planetary Computer, it has been added to the `SRW Data Bucket <https://registry.opendata.aws/noaa-ufs-shortrangeweather/>`_ and placed on :srw-wiki:`Level 1 <Supported-Platforms-and-Compilers>` systems in the usual input model data locations (see :numref:`Section %s <DataLocations>` for a list). Users can retrieve this data from the SRW App data bucket by running:
+
+.. code-block:: console
+
+   wget https://noaa-ufs-srw-pds.s3.amazonaws.com/develop-20240618/gulf_coast_blizzard.tar.gz
```

Comment:
This link is dead and I'm unable to find the associated gulf_coast_blizzard.tar.gz file on the updated S3 bucket.